README.md

```diff
@@ -164,7 +164,7 @@ or in `config/local.json`:
 ```
 
 Once you have config set,
-you can start the service with `cargo run`.
+you can start the service with `./r`.
```

Comment:
TBH I'm pretty "meh" on these tiny command aliases, they give me flashbacks to customs-server and all its tiny two-character JSON keys that are hard to understand without context.  But if you find them useful then I'm OK with it :-)

Reply:
I do find them useful fwiw. Partly because `cargo run --bin service` is annoying to type, but more because I expect I'll forget exactly what the invocation is any time I haven't run it for a few days.

Reply:
(to be clear in case it wasn't already, `cargo run` on it's own doesn't work any more)

Reply:
Wait, I may have misunderstood your comment. Are you meh because the script doesn't do much or me because the name is so short? I can give more readable names if it helps?

Reply:
The later, I'm specifically meh on the short opaque command name that a new reader probably won't understand when they check out the repo and run `ls`.

Reply:
The scripts now live under a `scripts/` directory with descriptive (hopefully) names. I've retained my opaquely-named helpers in the root as symlinks, because I genuinely do find them helpful.